### PR TITLE
feat(argo-rollouts): Add extra manifests option to chart

### PR DESCRIPTION
**Pull request by volym3ad**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Comment by volym3ad** [comment]
@mkilchhofer can you please review it?

File: charts/argo-rollouts/README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ If dashboard is installed by `--set dashboard.enabled=true`, checkout the argo-r
 | apiVersionOverrides.ingress | string | `""` | String to override apiVersion of ingresses rendered by this helm chart |
 | clusterInstall | bool | `true` | `false` runs controller in namespaced mode (does not require cluster RBAC) |
 | crdAnnotations | object | `{}` | Annotations to be added to all CRDs |
+| extraManifests | list | `[]` | Additional manifests to deploy within the chart. A list of objects. |
```
Comment (mkilchhofer) [inline charts/argo-rollouts/README.md:45]:
```suggestion
| extraObjects | list | `[]` | Additional manifests to deploy within the chart. A list of objects. |
```

File: charts/argo-rollouts/templates/extra-manifests.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{{ range .Values.extraManifests }}
```
Comment (mkilchhofer) [inline charts/argo-rollouts/templates/extra-manifests.yaml:1]:
```suggestion
{{ range .Values.extraObjects }}
```

File: charts/argo-rollouts/values.yaml [hunk 3]
```diff
@@ -22,6 +22,19 @@ apiVersionOverrides:
 # -- Override the Kubernetes version, which is used to evaluate certain manifests
 kubeVersionOverride: ""
 
+# -- Additional manifests to deploy within the chart. A list of objects.
+## Can be used to add secrets for Analysis with 3rd-party monitoring solutions.
+extraManifests: []
```
Comment (mkilchhofer) [inline charts/argo-rollouts/values.yaml:27]:
```suggestion
extraObjects: []
```

**Review by mkilchhofer — CHANGES_REQUESTED** [review]
While I'd prefer your proposed name, `extraObjects` is already implemented in the `argo-cd` helm chart. And I prefer consistency across our charts

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**